src/php/tests/interop/interop_client.php

```diff
@@ -251,6 +251,22 @@ function pingPong($stub) {
 }
 
 /**
+ * Run the empty_stream test.
+ * Passes when run against the Node server as of 2015-10-09
+ * @param $stub Stub object that has service methods.
+ */
+function emptyStream($stub) {
+  // for the current PHP implementation, $call->read() will wait
+  // forever for a server response if the server is not sending any.
+  // so this test is imeplemented as a timeout to indicate the absence
+  // of receiving any response from the server
+  $call = $stub->FullDuplexCall(array('timeout' => 100000));
+  hardAssert($call->read() === null, 'Server returned too many responses');
```

Comment:
Right, thanks for pointing it out. I also discovered that without calling `writesDone` the internal call will be messed up if I invoke the test a second time.